challenge.js

```diff
@@ -42,7 +43,7 @@
  */
 
 // ANSWER
-
+document.body.p or document.querySelector('p')
```

Comment:
We were specifically looking for the selector, but your answer is correct - `p` will select that particular element (among others).